core/bin/dev

```diff
@@ -12,7 +12,7 @@ else
 fi
 
 export NEXT_DEVELOPMENT_MODE=true
-export GROUPAROO_RUN_MODE="cli:start"
+export GROUPAROO_RUN_MODE="${GROUPAROO_RUN_MODE:='cli:start'}"
```

Comment:
This allows `npm run dev` to work in `apps/staging-config` again, and load the UI properly